dev/helpers.js

```diff
@@ -11,7 +11,7 @@ export function delay(ms) {
 export const glob = promisify(globModule);
 
 export function isMac() {
-  return os.platform === 'darwin';
+  return os.platform() === 'darwin';
```

Comment:
This is the actual problem we got with refactoring.

The retries are just an extra logic introduce while chasing the bug, kept it as a good to have.